README.md

```diff
@@ -321,37 +347,38 @@ When you create a workspace, `par` automatically:
 5. **Integrates with global par commands** - use `par ls`, `par open`, `par rm` etc.
 
 **Example directory structure:**
+
 ```
 # Original repositories anywhere on system:
-/home/user/projects/frontend/     # React app
-/home/user/projects/backend/      # Python API  
-/opt/company/docs/                # Documentation
+my-fullstack-app/
+├── frontend/           # React app
+├── backend/            # Python API
+└── docs/              # Documentation
 
 # After: par workspace start user-auth --repos /home/user/projects/frontend,/home/user/projects/backend,/opt/company/docs
 # Creates unified workspace at: ~/.local/share/par/workspaces/.../user-auth/
 ├── frontend/
-│   └── user-auth/     # Worktree with user-auth branch
 ├── backend/
-│   └── user-auth/     # Worktree with user-auth branch  
 ├── docs/
-│   └── user-auth/     # Worktree with user-auth branch
 └── user-auth.code-workspace
 
 # Single tmux session starts from workspace root
 # Navigate with: cd frontend/, cd backend/, cd docs/
-# Global session accessible via: par open user-auth (from anywhere)
+# Global session accessible via: par open user-auth
```

Comment:
Wonder if we should remove this section it's a bit confusing - not sure what it's trying to say with the directory structure